docs/modules/ROOT/pages/data-migration.adoc

```diff
@@ -1,39 +1,64 @@
-= Migrate Data to another Cluster
-:description: In this tutorial, you'll use xref:wan-replication.adoc[WAN Replication] to migrate data from one cluster to another.
+= Transfer Data to Another Cluster
+:description: In this tutorial, you'll transfer data from one cluster to another using xref:wan-replication.adoc[WAN Replication]. You'll also learn how to safely delete a cluster after the successful transfer of data.
+
 
 {description}
 
-== Before you Begin
+== Context
+
+WAN replication supports the following use cases:
 
-You cannot use WAN Replication if your cluster has developer mode turned on.
+* Migrate data to a new cluster before decommissioning your original cluster.
 
-You need at least two clusters with the same configuration. For example, if one cluster
-contains xref:custom-classes-upload.adoc[custom classes] or Persistence,
-you should enable and configure these options for the other cluster as well.
+* Use WAN replication with a xref:failover-clients-with-hazelcast-cloud.adoc[failover client] as part of your disaster recovery strategy. WAN replication automatically synchronizes data between clusters. If a cluster fails, the failover client can switch to an alternative cluster with minimal interruption to service.
+
+== Before you Begin
 
-== Step 1. Create WAN Replication
+* Set up two active clusters with developer mode turned off. Use the same configuration for each cluster. For example, if one cluster
+contains xref:custom-classes-upload.adoc[custom classes] or persistence,
+enable and configure the same options for the other cluster.
 
-In order to create WAN Replication, you can go to *Cluster Detail* > *Settings* > *Wan Replication* and click *Create New Replication*
+* Decide which cluster you want to:
+** Send or migrate data from. This is the _publisher_ cluster.
```

Comment:
It's confusing terminology, but we use 'active' and 'passive' to describe WAN clusters. This is slightly confusing because 'active' clusters can also be clusters in an active state.

Active-Passive: An active cluster replicates its maps and/or caches to one or more passive clusters for the purpose of maintaining a backup. If the active cluster becomes unavailable, you can redirect the application traffic to the passive cluster.

I prefer your terms. Maybe let's say 'also known as...'?

Reply:
Updated as suggested.